src/content/docs/iast/summary.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: New Relic IAST summary page
+tags:
+    - New Relic IAST
+    - IAST
+metaDescription: Find out the data you can find on the New Relic IAST summary page.
+---
+
+The New Relic IAST summary page provides a centralized view of your tested applications and your vulnerabilities at a glance. View vulnerabilities and their risk scores helps you to prioritize remediation decisions.
+
+Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > IAST** to open the New Relic IAST summary page.
+
+<img
```

Comment:
I worry that it's too difficult to parse this image. Maybe something like the labeled image here? https://docs.newrelic.com/docs/infrastructure/infrastructure-ui-pages/infra-ui-overview/